src/components/pages/PendingApplications/PendingApplication.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
+
+import { Table } from 'antd';
+import './PendingApplication.css';
+
+const columns = [
+  {
+    title: 'Name',
+    dataIndex: 'name',
+    defaultSortOrder: 'descend',
+    sorter: (a, b) => a.name - b.name,
+  },
+  {
+    title: 'Role',
+    dataIndex: 'role',
+    filters: [
+      {
+        text: 'mentor',
+        value: 'mentor',
+      },
+      {
+        text: 'mentee',
+        value: 'mentee',
+      },
+    ],
+    onFilter: (value, record) => record.role.includes(value),
+  },
+  {
+    title: 'Date',
+    dataIndex: 'date',
+    defaultSortOrder: 'descend',
+    sorter: (a, b) => a.date - b.date,
+  },
+];
+
+const PendingApplications = () => {
+  const [applications, setApplications] = useState([]);
+
+  useEffect(() => {
+    const getPendingApps = () => {
+      axiosWithAuth()
+        .get('http://localhost:8080/application')
```

Comment:
The only thing I'm seeing here is that we should be making use of our environment variables - by hardcoding `localhost` into this `get` request you've ensured that this won't work in deployment! 😱

Highly recommend making a patch to fix this as soon as you're free to!

Reply:
Thank you for pointing this out! I've pushed a commit that should fix this issue.

Reply:
@kennedybryant were you able to get this resolved?